packager/media/base/aes_encryptor.cc

```diff
@@ -125,15 +125,20 @@ bool AesCbcEncryptor::InitializeWithIv(const std::vector<uint8_t>& key,
   return SetIv(iv);
 }
 
+size_t AesCbcEncryptor::RequiredCiphertextSize(size_t plaintext_size) {
```

Comment:
I'm embarrassed for not thinking of this myself.  This is so much better!  We shouldn't have to have this knowledge distributed to all the call sites.  Thank you!